backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglPreferences.java

```diff
@@ -33,21 +33,38 @@
 import com.badlogic.gdx.utils.StreamUtils;
 
 public class LwjglPreferences implements Preferences {
-	private final String name;
+	private String name;
 	private final Properties properties = new Properties();
-	private final FileHandle file;
+	private FileHandle file;

```

Comment:
I also wonder here if there's any reason to hold onto an inaccessible `FileHandle` when it's only used in a constructor. I'm not sure what order my comments will show in, but I also mention this for Lwjgl3Preferences.